.github/workflows/winget.yml

```diff
@@ -0,0 +1,13 @@
+name: Publish to WinGet
+on:
+  release:
+    types: [released]
+jobs:
+  publish:
+    runs-on: windows-latest # action can only be run on windows
+    steps:
+      - uses: vedantmgoyal2009/winget-releaser@latest
```

Comment:
@mcraa should we pin this to a specific commit as discussed in the arch call?

Reply:
will do in #3780